### ISSUE TITLE
Blas/Lapack installation path

### ISSUE THREAD

**Issue by af1990** [GEOSX/thirdPartyLibs#34]
On Ubuntu 18.04, `blas` and `lapack` are installed in the `lib` directory, not `lib64` as required according to:
https://github.com/GEOSX/thirdPartyLibs/blob/e2ad476dd5b05e51207a21a8a78c3e274c64afa4/CMakeLists.txt#L100

**Comment by af1990** [comment]
I would change the main CMakeLists.txt as follows:

    # Check if system is using lib or lib64 as default library installation path
    get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
    set(LIBSUFFIX "")
    if (${LIB64} STREQUAL TRUE AND ${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
        set(LIBSUFFIX 64)
    endif()

    # Use the lapack_suite for trilinos. The last define is a hack.
    set(BLAS_LIBRARY_DIRS ${LAPACK_SUITE_DIR}/lib${LIBSUFFIX} CACHE PATH "")
    set(BLAS_LIBRARY_NAMES blas CACHE STRING "")

    set(LAPACK_LIBRARY_DIRS ${LAPACK_SUITE_DIR}/lib${LIBSUFFIX} CACHE PATH "")
    set(LAPACK_LIBRARY_NAMES lapack CACHE STRING "")

**Comment by af1990** [comment]
I tested it on both system (with lib and lib64 as default) and it works.
Both of them are Linux, I don't have a Mac for this test.

**Comment by rrsettgast** [comment]
@af1990 can you put your changes up in a PR?

**Comment by corbett5** [comment]
@af1990 I agree put them in a PR, but your proposed changes look good to me!

**Comment by af1990** [comment]
Now it is PR #36
